cfme/automate/explorer/method.py

```diff
@@ -472,6 +473,7 @@ def create(
         if validate and not BZ(1499881, forced_streams=['5.9']).blocks:
             add_page.validate_button.click()
             add_page.flash.assert_no_error()
+            add_page.wait_displayed()
```

Comment:
This should be above the flash message call to `assert_no_error` as well, just up one line please :)

Reply:
sure @mshriver